src/types.ts

```diff
@@ -9,6 +10,7 @@ type Worker = {
   shares: number;
   bestshare: number;
   bestever: number;
+  offline: boolean;
```

Comment:
Worker type represent the worker response from solock and should not be changed.
If you really need a new prop, you should extend this type and use the newly generated one. Example

```ts
type Person = {
  name: string
  age: number
}

type PersonWithSurname = Person & {
  surname: string
}
```
```